website/blog/2020-05-04-lithium.md

```diff
@@ -0,0 +1,312 @@
+compilation, which could make the Metals workspace hard to use. Our top priority
+is currently figuring out all the possible issues both in Metals as well in
+Bloop. Thanks to some fixes done by [jvican](https://github.com/jvican) we are
+hopeful that some of the issues might become less frequent. Those fixes are
+available in the current Bloop snapshots and in the version included in Metals
+
+We also reworked the Metals tree view to highlight some commands that a user can
+use to fix their workspace as well as added a new command to recompile workspace
+from scratch:
+
+![reworked-tree-view](https://i.imgur.com/PERfNqt.png)
+
+`Clean compile workspace` might be needed in case of broken Zinc caches and
+`Restart Bloop server` should help with the cases where deduplication of
+compilation would hang. Do not hesitate to report any possible issues and if
+possible there are a number of additional diagnostics that can be done before
+reporting an issue:
+
+- getting a thread dump of Bloop process
+- running verbose compilation via the CLI:
+  `coursier launch ch.epfl.scala:<bloop_version> -- compile --verbose <project_name>`
+- (advanced) attaching profiler such as YourKit to discover exceptions being
+  thrown by the Bloop server. This will require
+
+We are also monitoring the situation ourselves and investigating all problems
+encountered.
+
+## Automatically add '+' on newline inside a string
+
+The Metals team was recently joined by [mlachkar](https://github.com/mlachkar),
+an engineer from Scala Center, who already contributed a number of features and
+bug fixes to the Project. Thanks to her, we now can easily enter a newline
+inside a single line string and that string will be correctly split with an
+additional `+`.
+
+![split-line](https://i.imgur.com/uhF0MOx.gif)
+
+## Code action to import all missing symbols
+
+Previously, it was only available to import one symbol at a time in case of
+missing imports. Thanks to [colineto](https://github.com/colineto) we now have
+an ability to import everything in bulk for every unambiguous import available.
+Unambiguous in this case meaning that there is only one possible import to
+choose from.
+
+![import-all](https://i.imgur.com/mmzgJs7.gif)
+
+## Miscellaneous improvements
+
+- added an option to ignore .scalafmt.conf creation message
+- fixed hanging while debugging if workspace had errors
+- fixed issue with folding in non 2.12.11 workspaces
+- turn off "Go to parent" code lenses by default
+- fixed false parsing errors in 2.13.x workspaces
+- added date string to metals logs
+- fixed issue with additional src directories being created automatically
+- fixed "Go to implementation" to work with local aliases
+
+## Contributors
+
+Big thanks to everybody who contributed to this release! We had a lot of new
+contributors doing some really exemplary work.
+
+```
+$ git shortlog -sn --no-merges v0.8.4..v0.9.0
+Ayoub Benali
+Tomasz Godzik
+Scala Steward
+Chris Kipp
+Coline
```

Comment:
@colineto Do you want to be fully mentioned here with the surname or should I leave it as is?

Reply:
Coline Thomas will be great thx

Reply:
Updated!